app/controllers/cloud_volume_controller.rb

```diff
@@ -269,13 +269,8 @@ def new
     assert_privileges("cloud_volume_new")
     @volume = CloudVolume.new
     @in_a_form = true
-    @storage_manager_choices = {}
     if params[:storage_manager_id]
       @storage_manager_id = params[:storage_manager_id]
```

Comment:
@gberginc This line seems to be causing the Hakiri Security issue.
Is there any other way we could retrieve the `Storage Manager id` here without directly assigning `params[:storage_manager_id]` ?

I guess an alternate approach would be to loop through all EMSes that have `supports_block_storage? = true` and look for the one that matches `params[:storage_manager_id]`